requirements.txt

```diff
@@ -12,4 +12,5 @@ requests-futures==1.0.0
 requests==2.26.0
 sentry_sdk==1.4.3
 urllib3==1.26.7
-vdf==3.4
\ No newline at end of file
+vdf==3.4
+unixreg; sys_platform != 'win32'
```

Comment:
If having another external dependency is undesirable its possible to include the code in this repo

Reply:
It's fine if it's not big

Reply:
For reference
the source code of unixreg is available on PyPi and Github

https://pypi.org/project/unixreg/
https://github.com/Jan200101/unixreg